students/danwoj/Session12/trapz.py

```diff
@@ -0,0 +1,11 @@
+def trapz(fun, a, b):
+    '''
+    Trapezoidal Function
+    '''
+    x_values = [i for i in range(1,101)]
```

Comment:
Here's your issue -- you've got x from 0 to 100. You want 100 values, but you want them to be evenly spaced from a to b, not 0 to 100!

Reply:
Got it. Works as desired now. Thanks for the nudge.

Reply:
if you commit and push the change, it will show up in this PR, and I can then merge it.